examples/src/main/java/com/mapbox/navigation/examples/core/BaseRouterActivityJava.java

```diff
@@ -82,22 +80,12 @@ public static Router setupOffboardRouter(Context context) {
   }
 
   public static Router setupOnboardRouter(Context context) {
-    File file = new File(
-        Environment.getExternalStoragePublicDirectory("Offline").getAbsolutePath(),
```

Comment:
🔉  Here's our external storage option. I'll look into adding it back

Reply:
Going with the idea that, having the option to store router tiles externally is not supported.

Reply:
nevermind brought it back. now the example shows how to create external storage with the new builder